src/analysis/retail/monk/shared/TouchOfDeath.js

```diff
@@ -27,7 +27,7 @@ class TouchOfDeath extends ExecuteHelper {
     options.abilities.add({
       spell: SPELLS.TOUCH_OF_DEATH.id,
       category: SPELL_CATEGORY.COOLDOWNS,
-      cooldown: this.selectedCombatant.hasLegendary(SPELLS.FATAL_TOUCH) ? 60 : 180,
+      cooldown: 180,
```

Comment:
`cooldown: 180 - 45 * this.selectedCombatant.getTalentRank(TALENTS_MONK.FATAL_TOUCH_TALENT.id),`